com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeAssetEditor.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine.Experimental.Rendering;
+
+namespace UnityEditor.Experimental.Rendering
+{
+    [CustomEditor(typeof(ProbeVolumeAsset))]
+    class ProbeVolumeAssetEditor : Editor
```

Comment:
This might be unwanted? I personally really appreciate being able to see the internal stats of the baked data and I don't see the harm in having that visible.

Reply:
I think it is fine, a bit fringe use case is why it was not made.